docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '2'
+services:
+  ganache:
+    image: trufflesuite/ganache-cli:latest
+    ports:
+      - 8545:8545
+
+  kvstore:
+    build: .
+    image: hcaptcha/eth-kvstore
+    environment:
+      - ETH_HOST=ganache
+      - ETH_PORT=8545
+      - ADDRESS_OUTPUT_FILENAME=/deployed/ethkvstore.address.json
+    depends_on:
+      - ganache
+    volumes:
+      - deployed:/deployed
+
+  # I'll get rid of this, just wanna see it work
```

Comment:
As you mentioned in the comments, let's make sure this one is removed before merging.